aprslib/parsing/thirdparty.py

```diff
@@ -0,0 +1,21 @@
+import re
+from aprslib.parsing.__init__ import parse
+from aprslib.exceptions import UnknownFormat
+from aprslib.exceptions import ParseError
+
+__all__ = [
+        'parse_thirdparty',
+        ]
+
+def parse_thirdparty(body):
+    parsed = {'format':'Third party'}
+
+    # Parse sub-packet
+    try:
+        subpacket = parse(body)
+    except (UnknownFormat) as ukf:
```

Comment:
Should add this patch as well,
https://github.com/wsmitchell3/aprs-python/commit/441edbfeedf477bd307a622b572b525b181266b2